


## Protein Folding Resource Collection


In this repository you will find papers, tutorials and open-resource projects related to protein folding. 
The goal is to collect learning resources.
This repository is contribution friendly, so if you want to add something, please feel free to submit a PR.


### General Protein Folding


- Peng, Zhenling, Wenkai Wang, Hong Wei, Xiaoge Li, and Jianyi Yang. 
[Improved protein structure prediction with trRosettaX2, AlphaFold2, and optimized MSAs in CASP15](http://yanglab.qd.sdu.edu.cn/papers/Peng_Proteins_2023.pdf). 
Proteins: Structure, Function, and Bioinformatics. 2023.


- Baek, Minkyung, Ivan Anishchenko, Ian Humphreys, Qian Cong, David Baker, and Frank DiMaio. 
[Efficient and accurate prediction of protein structure using RoseTTAFold2](https://www.biorxiv.org/content/10.1101/2023.05.24.542179.full.pdf).
bioRxiv (2023): 2023-05.
**RoseTTAFold2**


- Terwilliger, Thomas C., Billy K. Poon, Pavel V. Afonine, Christopher J. Schlicksup, Tristan I. Croll, Claudia Millán, Jane S. Richardson, Randy J. Read, and Paul D. Adams. 
[Improved AlphaFold modeling with implicit experimental information](https://www.nature.com/articles/s41592-022-01645-6). 
Nature methods 19, no. 11 (2022): 1376-1382.


- Xiaomin Fang, Fan Wang, Lihang Liu, Jingzhou He, Dayong Lin, Yingfei Xiang, Xiaonan Zhang, Hua Wu, Hui Li, Le Song.
[HelixFold-Single: MSA-free Protein Structure Prediction by Using Protein Language Model as an Alternative](https://arxiv.org/pdf/2207.13921.pdf). 
arXiv. 2022.
**HelixFold-Single**


- Lin, Zeming, Halil Akin, Roshan Rao, Brian Hie, Zhongkai Zhu, Wenting Lu, Allan dos Santos Costa et al. 
[Evolutionary-scale prediction of atomic level protein structure with a language model](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v2.full.pdf). 
bioRxiv (2022).
**ESMFold**



- Wu, Ruidong, Fan Ding, Rui Wang, Rui Shen, Xiwen Zhang, Shitong Luo, Chenpeng Su et al. 
[High-resolution de novo structure prediction from primary sequence](https://www.biorxiv.org/content/10.1101/2022.07.21.500999.full.pdf).
BioRxiv (2022).
**OmegaFold**


- Robin Pearce,Yang Li,Gilbert S. Omenn,Yang Zhang.
[Fast and accurate Ab Initio Protein structure prediction using deep learning potentials](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010539)
PLOS Computational Biology. 2022.
**DeepFold**



- Barrett, Thomas D., Amelia Villegas-Morcillo, Louis Robinson, Benoit Gaujac, David Admete, Elia Saquand, Karim Beguir, and Arthur Flajolet. 
[So ManyFolds, So Little Time: Efficient Protein Structure Prediction With pLMs and MSAs](https://www.biorxiv.org/content/10.1101/2022.10.15.511553.full.pdf).
bioRxiv (2022): 2022-10. 
**MonoFold/PolyFold**


- Chowdhury, Ratul, Nazim Bouatta, Surojit Biswas, Charlotte Rochereau, George M. Church, Peter K. Sorger, and Mohammed AlQuraishi. 
[Single-sequence protein structure prediction using language models from deep learning](https://www.biorxiv.org/content/biorxiv/early/2021/08/04/2021.08.02.454840.full.pdf). 
bioRxiv (2021).
**RGN2**

  
- Evans, Richard, Michael O’Neill, Alexander Pritzel, Natasha Antropova, Andrew Senior, Tim Green, Augustin Žídek et al. 
[Protein complex prediction with AlphaFold-Multimer](https://www.biorxiv.org/content/biorxiv/early/2022/03/10/2021.10.04.463034.full.pdf). 
BioRxiv (2022): 2021-10.
**AlphaFold-Multimer**


- Jumper, John, Richard Evans, Alexander Pritzel, Tim Green, Michael Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool et al. 
[Highly accurate protein structure prediction with AlphaFold](https://www.nature.com/articles/s41586-021-03819-2).
Nature 596, no. 7873 (2021): 583-589.
**AlphaFold2**


- Wu, Jiaxiang, Tao Shen, Haidong Lan, Yatao Bian, and Junzhou Huang. 
[SE (3)-equivariant energy-based models for end-to-end protein folding](https://www.biorxiv.org/content/10.1101/2021.06.06.447297.full.pdf).
bioRxiv (2021): 2021-06.
**SE(3)-Fold**


- Yang, Jianyi, Ivan Anishchenko, Hahnbeom Park, Zhenling Peng, Sergey Ovchinnikov, and David Baker. 
[Improved protein structure prediction using predicted interresidue orientations](https://www.pnas.org/doi/full/10.1073/pnas.1914677117). 
Proceedings of the National Academy of Sciences (PNAS) 117, no. 3 (2020): 1496-1503.
**trRosetta**


- AlQuraishi, Mohammed. 
[End-to-end differentiable learning of protein structure](https://www.cell.com/cell-systems/pdf/S2405-4712(19)30076-6.pdf).
Cell systems 8, no. 4 (2019): 292-301.


- Xu, Jinbo. 
[Distance-based protein folding powered by deep learning](https://www.pnas.org/doi/full/10.1073/pnas.1821309116).
Proceedings of the National Academy of Sciences 116, no. 34 (2019): 16856-16865.


- Adhikari, Badri, Debswapna Bhattacharya, Renzhi Cao, and Jianlin Cheng. 
[CONFOLD: residue‐residue contact‐guided ab initio protein folding](https://www.academia.edu/download/45754134/Adhikari_et_al-2015-Proteins-_Structure_20160518-23277-apohcs.pdf)
Proteins: Structure, Function, and Bioinformatics 83, no. 8 (2015): 1436-1449.
**CONFOLD**


- Brunger, Axel T. 
[Version 1.2 of the Crystallography and NMR system](https://www.mrc-lmb.cam.ac.uk/public/xtal/doc/cns/cns_1.3/about_cns/brunger_nature_protocols_2007.pdf).
Nature protocols 2, no. 11 (2007): 2728-2733.
**CNS** (a useful tool to determine structure from distance map.)



### Antibody/Nanobody/Loop/Mini-Protein Folding

- Burbach, Sarah M., and Bryan Briney. 
[Improving antibody language models with native pairing](https://arxiv.org/pdf/2308.14300).
arXiv preprint arXiv:2308.14300 (2023).


- Fernández-Quintero, Monica L., Janik Kokot, Franz Waibl, Anna-Lena M. Fischer, Patrick K. Quoika, Charlotte M. Deane, and Klaus R. Liedl. 
[Challenges in antibody structure prediction](https://www.tandfonline.com/doi/full/10.1080/19420862.2023.2175319). 
In MAbs, vol. 15, no. 1, p. 2175319. Taylor & Francis, 2023.


- Ruffolo, Jeffrey A., and Jeffrey J. Gray. 
[Fast, accurate antibody structure prediction from deep learning on massive set of natural antibodies](https://www.nature.com/articles/s41467-023-38063-x). 
Nature communications 14, no. 1 (2023): 2389.
**IgFold**


- Yining Wang, Xumeng Gong, Shaochuan Li, Bing Yang, Yiwu Sun, Yujie Luo, Hui Li, Le Song
[Fast de novo antibody structure prediction with atomic accuracy](https://aacrjournals.org/cancerres/article/83/7_Supplement/4296/722468).
The American Association for Cancer Research. 4296-4296. 2023.
**xTrimoABFold++**


- Peng, Chao, Zelong Wang, Peize Zhao, Weifeng Ge, and Charles Huang. 
[AbFold--an AlphaFold Based Transfer Learning Model for Accurate Antibody Structure Prediction](https://www.biorxiv.org/content/10.1101/2023.04.20.537598.full.pdf).
bioRxiv (2023): 2023-04.
**AbFold**


- Kończak, Jarosław, Bartosz Janusz, Jakub Młokosiewicz, Tadeusz Satława, Sonia Wróbel, Paweł Dudzic, and Konrad Krawczyk. 
[Structural pre-training improves physical accuracy of antibody structure prediction using deep learning](https://www.sciencedirect.com/science/article/pii/S2667119023000083).
ImmunoInformatics (2023): 100028.


- Wang, Junlin, Wenbo Wang, and Yi Shang. 
[Protein Loop Modeling Using AlphaFold2](https://ieeexplore.ieee.org/abstract/document/10098730/).
IEEE/ACM Transactions on Computational Biology and Bioinformatics (2023).
**IAFLoop**


- Wu, Jiaxiang, Fandi Wu, Biaobin Jiang, Wei Liu, and Peilin Zhao. 
[tFold-ab: fast and accurate antibody structure prediction without sequence homologs](https://www.biorxiv.org/content/10.1101/2022.11.10.515918.full.pdf).
bioRxiv (2022): 2022-11.
**tFold-ab**


- Tripathi, Asmita, Rajkrishna Mondal, Tapobrata Lahiri, Deepak Chaurasiya, and Manoj Kumar Pal. 
[TemPred: A novel protein template search engine to improve protein structure prediction](https://www.computer.org/csdl/journal/tb/5555/01/10005099/1JDoTyLTEaY).
IEEE/ACM Transactions on Computational Biology and Bioinformatics 01 (2023): 1-12.
**TemPred**


- Giovanoudi, Eleni, and Dimitrios Rafailidis. 
[Multi-Task Learning with Loop Specific Attention for CDR Structure Prediction](https://arxiv.org/pdf/2306.13045.pdf). 
arXiv preprint arXiv:2306.13045 (2023).


- Wang, Tianyue, Xujun Zhang, Odin Zhang, Peichen Pan, Guangyong Chen, Yu Kang, Chang-Yu Hsieh, and Tingjun Hou. 
[Highly accurate and efficient deep learning paradigm for full-atom protein loop modeling with KarmaLoop](https://arxiv.org/pdf/2306.12754). 
arXiv preprint arXiv:2306.12754 (2023).
**KarmaLoop**


- Ruffolo, Jeffrey A., Jeremias Sulam, and Jeffrey J. Gray. 
[Antibody structure prediction using interpretable deep learning](https://www.cell.com/patterns/pdf/S2666-3899(21)00280-4.pdf). 
Patterns 3, no. 2 (2022).


- Abanades, Brennan, Guy Georges, Alexander Bujotzek, and Charlotte M. Deane. 
[ABlooper: Fast accurate antibody CDR loop structure prediction with accuracy estimation](https://academic.oup.com/bioinformatics/article/38/7/1877/6517780).
Bioinformatics 38, no. 7 (2022): 1877-1880.
**ABlooper**


- Abanades, Brennan, Wing Ki Wong, Fergus Boyles, Guy Georges, Alexander Bujotzek, and Charlotte Mary Deane. 
[ImmuneBuilder: Deep-Learning models for predicting the structures of immune proteins](https://www.biorxiv.org/content/10.1101/2022.11.04.514231v1.full.pdf).
bioRxiv (2022).
**ImmuneBuilder**


- Cohen, Tomer, Matan Halfon, and Dina Schneidman-Duhovny.
[NanoNet: Rapid and accurate end-to-end nanobody modeling by deep learning](https://www.frontiersin.org/articles/10.3389/fimmu.2022.958584/pdf). 
Frontiers in immunology 13 (2022): 958584.
**NanoNet**


- Wang, Yining, Xumeng Gong, Shaochuan Li, Bing Yang, YiWu Sun, Chuan Shi, Hui Li, Yangang Wang, Cheng Yang, and Le Song. 
[xtrimoABFold: Improving antibody structure prediction without multiple sequence alignments](https://arxiv.org/pdf/2212.00735v3.pdf). 
arXiv preprint arXiv:2212.00735 (2022).
**xTrimoABFold**


- Lee, Jae Hyeon, Payman Yadollahpour, Andrew Watkins, Nathan C. Frey, Andrew Leaver-Fay, Stephen Ra, Kyunghyun Cho, Vladimir Gligorijevic, Aviv Regev, and Richard Bonneau. 
[EquiFold: protein structure prediction with a novel coarse-grained structure representation](https://www.biorxiv.org/content/10.1101/2022.10.07.511322.full.pdf).
bioRxiv (2022): 2022-10.
**EquiFold**


- Ruffolo, Jeffrey A., Jeremias Sulam, and Jeffrey J. Gray. 
[Antibody structure prediction using interpretable deep learning](https://www.sciencedirect.com/science/article/pii/S2666389921002804).
Patterns 3, no. 2 (2022): 100406.


### Protein Conformation/Distribution

- Jing, Bowen, Ezra Erives, Peter Pao-Huang, Gabriele Corso, Bonnie Berger, and Tommi Jaakkola. 
[EigenFold: Generative Protein Structure Prediction with Diffusion Models](https://arxiv.org/pdf/2304.02198). 
arXiv preprint arXiv:2304.02198 (2023).



### Engineering Tech on Folding

- Lee, Jaemyung, Jaehoon Kim, Hasun Yu, and Youhan Lee. 
[Solvent: A Framework for Protein Folding](https://arxiv.org/pdf/2307.04603).
arXiv preprint arXiv:2307.04603 (2023).


- Mirdita, Milot, Konstantin Schütze, Yoshitaka Moriwaki, Lim Heo, Sergey Ovchinnikov, and Martin Steinegger. 
[ColabFold: making protein folding accessible to all](https://www.nature.com/articles/s41592-022-01488-1).
Nature Methods (2022): 1-4.
**ColabFold**


- Li, Ziyao, Xuyang Liu, Weijie Chen, Fan Shen, Hangrui Bi, Guolin Ke, and Linfeng Zhang.
  [Uni-Fold: An Open-Source Platform for Developing Protein Folding Models beyond AlphaFold](https://www.biorxiv.org/content/10.1101/2022.08.04.502811v3.full.pdf). 
bioRxiv (2022).
**Uni-Fold**


- Cheng, Shenggan, Ruidong Wu, Zhongming Yu, Binrui Li, Xiwen Zhang, Jian Peng, and Yang You.
  [FastFold: Reducing AlphaFold Training Time from 11 Days to 67 Hours](https://arxiv.org/pdf/2203.00854.pdf). 
arXiv preprint arXiv:2203.00854 (2022).
**FastFold**

- Ahdritz, Gustaf, Nazim Bouatta, Sachin Kadyan, Qinghui Xia, William Gerecke, Timothy J. O’Donnell, Daniel Berenberg et al. 
[OpenFold: Retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization](https://www.biorxiv.org/content/10.1101/2022.11.20.517210.full.pdf). 
bioRxiv (2022): 2022-11.

- Wang, Guoxia, Xiaomin Fang, Zhihua Wu, Yiqun Liu, Yang Xue, Yingfei Xiang, Dianhai Yu, Fan Wang, and Yanjun Ma. 
[Helixfold: An efficient implementation of alphafold2 using paddlepaddle](https://arxiv.org/pdf/2207.05477).
arXiv preprint arXiv:2207.05477 (2022).


### Related Works

- Kandathil, Shaun M., Andy M. Lau, and David T. Jones. 
[Machine learning methods for predicting protein structure from single sequences](https://www.sciencedirect.com/science/article/pii/S0959440X2300101X).
Current Opinion in Structural Biology 81 (2023): 102627.

